src/runtime/HalideRuntimeHexagonHost.h

```diff
@@ -0,0 +1,66 @@
+#ifndef HALIDE_HALIDERUNTIMEHEXAGONHOST_H
+#define HALIDE_HALIDERUNTIMEHEXAGONHOST_H
+
+#include "HalideRuntime.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** \file
+ *  Routines specific to the Halide Hexagon host-side runtime.
+ */
+
+typedef int halide_hexagon_handle_t;
+
+extern const struct halide_device_interface *halide_hexagon_device_interface();
+
+/** Set the underlying ion device handle for a buffer. The device
+ * handle should be allocated using ion_alloc or similar and must
```

Comment:
Normally, in https://android.googlesource.com/platform/system/core/+/master/libion/include/ion/ion.h. In Halide, we have a "mini_ion.h", similar to the other "mini_*" headers.